taxi_service/settings.py

```diff
@@ -139,6 +141,9 @@
 
 STATIC_ROOT = BASE_DIR / "staticfiles"
 
+CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap4"
+CRISPY_TEMPLATE_PACK = "bootstrap4"
```

Comment:
The setting 'CRISPY_TEMPLATE_PACK' is defined twice. You should remove one of these duplicate lines to avoid confusion or potential issues.